site/content/en/docs/Guides/metrics.md

```diff
@@ -207,9 +214,29 @@ kubectl port-forward deployments/grafana 3000 -n metrics
 
 Open a web browser to [http://localhost:3000](http://localhost:3000), you should see Agones [dashboards](#grafana-dashboards) after login as admin.
 
-### Cloud Monitoring installation
+{{% feature publishVersion="1.30.0" %}}
+### Google Cloud Managed Service for Prometheus installation
+
+To collect Agones metrics using [Managed Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus):
+
+* [Enable managed collection](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#enable-mgdcoll-gke) on your cluster (or [non-GKE](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#enable-mgdcoll-non-gke)).
+* Configure Managed Prometheus to scrape Agones by configuring a `PodMonitoring` resource:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/build/prometheus-google-managed.yaml
+```
+* At this point, you should be able to see [Prometheus metrics in Cloud Monitoring](https://cloud.google.com/monitoring/promql). If that's all you need, you can stop here.
+
+To install Grafana using a Managed Prometheus backend:
+
+* Install the [Standalone Prometheus frontend UI](https://cloud.google.com/stackdriver/docs/managed-prometheus/query#ui-prometheus) - this will act as your authentication proxy for PromQL queries. To group it together with our [Grafana installation][#grafana-installation], install it in the `metrics` namespace.
```

Comment:
The link for the grafana installation didn't render properly. Needs () instead of [] around the second part.

Reply:
Oops, fixed.